### PR TITLE
Signing in edge cases

### PR DESCRIPTION
**Pull request by EdwardAndress**
##One failing test just added
* signing_in.feature
* The user signs in then enters '/admin' url
* They should be redirected to the search page for their tribunal
* But there is a [GET] 'decisions' route not found error
 * Also fails if logged in as admin+tax at http://tribunals-dev.is.dsd.io/

- Please let me know how you solve this... or how I can solve it! :)

### CODE REVIEW

File: features/signing_out.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+   When I click "Sign out"
+   Then I click "No, take me back"
+   Then I should be sent back to the page where I first clicked sign out
+
+  Scenario: Clicked 'Sign out' twice, then clicked 'No, take me back'
+   Given I am in the process of adding a new decision
+   When I click "Sign out"
+   Then I click "Sign out"
+   Then I click "No, take me back"
+   Then I should be sent back to the page where I first clicked sign out
```
Comment (tatyree) [inline features/signing_out.feature]: No route matches [GET] "decisions" (ActionController::RoutingError)
  ./features/step_definitions/signing_in_steps.rb:50:in `/^I enter the url '\/admin'$/'
  features/signing_in.feature:66:in `When I enter the url '/admin''

File: features/signing_in.feature [hunk 2]
```diff
@@ -54,4 +54,14 @@ Feature: Signing in
...
+    And I remain on the sign in page
+
+    Scenario: Sign in then enter /admin url
+    When I go to the sign in page
+     And I enter an existing user's email address
+     And I enter a valid password
+     And I click 'Sign in'
+     And I am sent to the front-end decisions page for my tribunal
+     And I am on the upload document page
+    When I enter the url '/admin'
```
Comment (tatyree) [inline features/signing_in.feature]: No route matches [GET] "decisions" (ActionController::RoutingError)
  ./features/step_definitions/signing_in_steps.rb:50:in `/^I enter the url '\/admin'$/'
  features/signing_in.feature:66:in `When I enter the url '/admin''

**Comment by tatyree** [comment]
Please fix the two failing specs and then I'll merge. See notes...
